proto/harpocrate.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+package gravitalia.harpocrate.v1;
+
+option csharp_namespace = "Gravitalia.Harpocrate.V1";
+option go_package = "github.com/gravitalia/harpocrate/proto";
+option java_package = "com.gravitalia.harpocrate.v1";
+option java_multiple_files = true;
+option java_outer_classname = "HarpocrateProto";
+
+// The shortener service creation that allows to reduce URL length
+service Harpocrate {
+    // Reduce link length by according a random ID to a link
+    rpc Reduce (ReduceRequest) returns (ReduceReponse) {}

```

Comment:
If the RPC does not contain `option`s, make the line end with a semicolon instead of empty brackets:

```diff
-rpc Reduce (ReduceRequest) returns (ReduceReponse) {}
+rpc Reduce (ReduceRequest) returns (ReduceReponse);
```